governance/roles.md

```diff
@@ -283,4 +289,37 @@ Some ongoing projects may have teams where members are identified for
 additional roles and may be required to have specific expertise. For visibility,
 these additional project roles are listed below:
 
-* [Security Reviewers](../assessments/guide/security-reviewer.md/)
+## Lead rotations
+
+For certain facilitator roles, teams and projects that are ongoing, leadership
+rotations should be carried out every 6 months. This is to:
+
+* Ensure leadership continuity of projects and passing on of tribal knowledge
+* Ensure diverse perspective/breadth of knowledge
+* Prevention of commitment bias in projects
+* Provide more leadership opportunities for more community members
+
+A facilitator role/project lead position should be a candidate for rotation if the
+project/facilitation task:
+
+* Exists for more than 6 months
+* Involves more than 1 person
+
+### Rotation process
+
```

Comment:
Timing of the rotation will be decided by STAG Co-Chairs for respective projects. Validation that proper transition has happened will be done by STAG representative.

Reply:
>Validation that proper transition has happened will be done by STAG representative.

added the point on validation to the paragraph below.

>Timing of the rotation will be decided by STAG Co-Chairs for respective projects.

I think that having the rotation process be triggered every X timeframe is important, as it provides a means to check-in. Since a rotation process doesn't necessary need to result in a rotation. I do see there being an issue of overhead, so perhaps we could have rotations be done on a 6 monthly basis on certain dates, like May 30, Nov 30. That would also make staffing and spreading opportunities for leadership better as well. Each STAG rep would get the rotation info from each project they rep and we can just do a one time sit-down to go through all the projects.